src/utils/__tests__/paths.spec.ts

```diff
@@ -59,6 +59,11 @@ describe('paths', () => {
       darwin: new Error('wslDistroData'),
     },
     lima: {
+      win32:  new Error('lima'),
+      linux:  '%HOME%/.rd/lima/',
+      darwin: '%HOME%/.rd/lima/',
+    },
+    oldLima: {
       win32:  new Error('lima'),
```

Comment:
Doesn't really matter, but for consistency this should be updated too:

```suggestion
      win32:  new Error('oldLima'),
```